service-node/frontend/src/main/scala/org/enterprisedlt/fabric/service/node/connect/ServiceNodeRemote.scala

```diff
@@ -159,5 +199,6 @@ object ServiceNodeRemote {
           .map(_.responseText)
           .map(r => upickle.default.read[Events](r))
 
-
+    private def mkURLGet(url: String, parameters: (String, String)*): String =
+        s"$url${parameters.map(v => s"${v._1}=${v._2}").mkString("?", "&", "")}"
```

Comment:
the URL encoding for parameters is require here

Reply:
removed sine it is unused.